dev-libs/aws-c-common/Manifest

```diff
@@ -1 +1 @@
-DIST aws-c-common-0.4.11.tar.gz 306933 BLAKE2B 5a2370a3696e4d90b67bc03fbf88f152e1a2d6e4d4a9ff2a9423a2cf18c14a1e3b6d978250c3359f4ebb2cf5e524e416b11604dc09dc45f5c8a5a6f982b344a7 SHA512 1e00f6aa025578978551fc8ca1bc09b448f79ac28bdc969af4312569ffca035d656dc68825da15ad8f4f890bbdb03327c624147dded1ce402e9dab1726290890
+DIST aws-c-common-0.4.57.tar.gz 418954 BLAKE2B cfb268d92e90343e01756b571d0d7fa1269a144b933b5dfae6e5fb92a2b384ff73bf14ad830b53427ad37e9dbd9a0afff5b571108869a69a8d466982a2e6c6ef SHA512 77a9110a067e3057ae3cb90ee0b1fa42e18595e476b803f83711ecc164053db73683df2fccf75ad992b7d08e142cb31d6a70f87ce2f3b87ce28ee1e8151b3c62
```

Comment:
FYI This should be 2 commits, one for adding a new version, 2nd for removing old.